### DISCUSSION TITLE
깨진 강의자료 요청

### DISCUSSION THREAD

**Discussion by khg2478** [yoonjeong-kwon/fastcampus-kubernetes#10] [Q&A] [answered]
안녕하세요 권윤정 강사님,
우선 좋은 강의자료를 만들어주신점에 대해 감사합니다.

part3.-쿠버네티스-입문-ch01-05-pdf- 파일에 깨진 파일들이 많아 다운로드가 어렵습니다.
ch01, ch04, ch05 3가지 자료들이 압축해제가 불가한 상태인데 파일 전송 좀 부탁드려도 될까요?
khg2478@gmail.com 으로 부탁 좀 드리겠습니다.

감사합니다.

**Comment by yoonjeong-kwon** [comment, marked as answer]
안녕하세요.^^
자료는 메일로 전송해드렸습니다.
확인 부탁드리겠습니다.

감사합니다!
